src/biogeophys/SnowSnicarMod.F90

```diff
@@ -433,7 +698,8 @@ subroutine SNICAR_RT (flg_snw_ice, bounds, num_nourbanc, filter_nourbanc,  &
                sfctype           = -1
```

Comment:
...investigate and add comment.

Reply:
@ekluzek this section sets variables (including sfctype) specific to CSIM. I also see a flag `flg_snw_ice` that equals 1 when calling this routine from CLM and 0 when calling this routine from CSIM.

@cenlinhe is it still an option to call this code from CSIM rather than CLM or should I remove these lines?

Reply:
@ekluzek:
if we wanted to run with CSIM, we would need infrastructure in place in the repository, plus testing that it works. So recommendation is to remove.